components/InventoryItemListItem/index.tsx

```diff
@@ -0,0 +1,173 @@
+import { TableRow, TableCell, Chip, Button } from '@mui/material'
+import { InventoryItem } from 'utils/types'
+import WarningIcon from '@mui/icons-material/Warning'
+import { MoreVert } from '@mui/icons-material'
+import theme from 'utils/theme'
+
+interface InventoryItemListItemProps {
+  inventoryItem: InventoryItem
+}
+
+export default function InventoryItemListItem({
+  inventoryItem,
+}: InventoryItemListItemProps) {
+  // ensure that attribute chips are readable
+  const getContrastYIQ = (hexcolor: string) => {
+    var colorNoHash = hexcolor.substring(1, hexcolor.length - 1)
+    var r = parseInt(colorNoHash.substring(0, 2), 16)
+    var g = parseInt(colorNoHash.substring(2, 4), 16)
+    var b = parseInt(colorNoHash.substring(4, 2), 16)
+    var yiq = (r * 299 + g * 587 + b * 114) / 1000
+    return yiq >= 128 ? 'black' : 'white'
+  }
+
+  // needed to deal with the possible union types defined in the database schema
+  const narrowCategory = (inventoryItem: InventoryItem) => {
+    if (
+      typeof inventoryItem.itemDefinition === 'string' ||
+      !inventoryItem.itemDefinition.category
+    ) {
+      return ' '
+    }
+
+    if (typeof inventoryItem.itemDefinition.category === 'string') {
+      return inventoryItem.itemDefinition.category
+    }
+
+    return inventoryItem.itemDefinition.category.name
+  }
+
+  // renders the red or yellow warning symbol if necesary
+  const renderWarningIcon = (inventoryItem: InventoryItem) => {
```

Comment:
@andlrutt when I hover over the warning icon I am kind of expecting so text to pop up in a tooltip. I think we should add this with a message that's like `This item has [critically low|low] stock`.  This may be something to come back to though. If we do want to do it, but later, we should add a todo comment here.

Reply:
Done with 92141b4 and the MUI Tooltip component